iothub/device/src/TelemetryMethods.cs

```diff
@@ -26,7 +26,10 @@ public static string GetSqmMachineId()
                 catch (Exception ex)
                 {
                     Debug.Assert(false, ex.Message);
-                    if (Logging.IsEnabled) Logging.Error(null, ex, nameof(TelemetryMethods));
```

Comment:
Isn't a single statement allowed to be without enclosing blocks?

Reply:
Allowed by the language, but not preferred by the style. I'm going to revert this change for now, because I plan to put the condition in the method instead.